python/lsst/ts/watcher/rules/mtm2_force_error.py

```diff
@@ -0,0 +1,193 @@
+# This file is part of ts_watcher.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["MTM2ForceError"]
+
+import logging
+import types
+import typing
+
+import yaml
+from lsst.ts import salobj, watcher
+from lsst.ts.xml.enums.Watcher import AlarmSeverity
+
+from ..base_rule import AlarmSeverityReasonType, NoneNoReason
+from ..remote_info import RemoteInfo
+
+
+class MTM2ForceError(watcher.PollingRule):
```

Comment:
Are you sure you want to do a `PollingRule` for this one? I feel like a normal rule acting asynchronously would be better no?

Reply:
This reason I used the poll is based on:

https://ts-watcher.lsst.io/writing_rules.html#constructor

```
High bandwidth telemetry: always poll, to avoid overwhelming the Watcher.
```

The telemetry rate of M2 is 20 Hz now.